### PR TITLE
Add logql basics

### PR DESCRIPTION
**Pull request by QuentinBisson**
### What this PR does / why we need it

Closes https://github.com/giantswarm/giantswarm/issues/29777

### Things to check/remember before submitting

- If you made content changes

    - Run `make dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

@giantswarm/team-atlas can you pull it and run make dev to check the page?

### CODE REVIEW

File: src/content/vintage/getting-started/observability/visualization/log-exploration/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+### Filter operators
+
+Used to filter text within the log stream
+
+```nohighlight
+|= : contains
+!= : does not contain
+|~ : regex matches
+!~ : regex does not match
```
Comment (pipo02mix) [inline src/content/vintage/getting-started/observability/visualization/log-exploration/index.md:68]:
```suggestion
Filter operators are used to refine the selection of text within the log stream. They help you narrow down the results by matching specific patterns or excluding certain data.

```nohighlight
|= : This operator is used when the log stream contains a specific string. It returns all log lines that contain the specified string.
!= : This operator is used when the log stream does not contain a specific string. It filters out and excludes all log lines that contain the specified string.
|~ : This operator is used for regex (regular expression) matching. It returns all log lines where the specified regular expression pattern matches.
!~ : This operator is used when the regular expression does not match the log stream. It filters out and excludes all log lines where the specified regular expression pattern matches.
```

File: src/content/vintage/getting-started/observability/visualization/log-exploration/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+### Log stream selectors
+
+Used to filter the log stream by labels
+
+```nohighlight
+=  : equals
+!= : not equals
+=~ : regex matches
+!~ : regex does not match
```
Comment (pipo02mix) [inline src/content/vintage/getting-started/observability/visualization/log-exploration/index.md:57]:
```suggestion
Log stream selectors are used to filter the log stream by labels. Labels are key-value pairs that can be attached to each log stream. These selectors are used to find specific log streams where the labels match certain conditions.

```nohighlight
=  : This operator checks for label values that are exactly equal to the specified value. It returns all log streams where the label value equals the specified value.

!= : This operator checks for label values that are not equal to the specified value. It returns all log streams where the label value does not equal the specified value.

=~ : This operator is used for regex (regular expression) matching on label values. It returns all log streams where the label value matches the specified regular expression pattern.

!~ : This operator is used when the regular expression does not match the label value. It returns all log streams where the label value does not match the specified regular expression pattern.
```

File: src/content/vintage/getting-started/observability/visualization/log-exploration/index.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+last_review_date: 2024-02-26
+user_questions:
+  - How can I access logs from my installation?
+  - Where can I find workload cluster logs?
+  - Which logs are gathered in my installation?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+Logs, which are records of events and activities generated by systems and applications, serve as a rich source of information for administrators, developers, and security professionals. The ability to search through logs is essential to help with troubleshooting and debugging by looking through a detailed history of events, errors and warnings. It is also a valuable source of information in context of performance monitoring, security and compliance.
```
Comment (pipo02mix) [inline src/content/vintage/getting-started/observability/visualization/log-exploration/index.md:21]:
```suggestion
Logs, which are records of events and activities generated by systems and applications, serve as a rich source of information for administrators, developers, and security professionals. The ability to search through logs is essential to help with troubleshooting and debugging by looking through a detailed history of events, errors and warnings. It is also a valuable source of information in the context of performance monitoring, security and compliance.
```

File: src/content/vintage/getting-started/observability/visualization/log-exploration/index.md (hunk 2 above)
Comment (QuentinBisson) [reply src/content/vintage/getting-started/observability/visualization/log-exploration/index.md:57]: Do you think we still need the ``` block then?

File: src/content/vintage/getting-started/observability/visualization/log-exploration/index.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+3. Choose how you prefer to build your queries:
+   * `builder` and play with the dropdowns to build your query
+   * `code` to write your query using [LogQL](https://grafana.com/docs/loki/latest/logql/)
+
+__Note__: The live mode feature of Grafana Loki is not available at the moment because we use multi-tenancy (c.f. https://github.com/grafana/loki/issues/9493) 😢
+
+## LogQL basics
+
+Here are a few LogQL basics to help you get started.
```
Comment (pipo02mix) [inline src/content/vintage/getting-started/observability/visualization/log-exploration/index.md:43]:
```suggestion
[LogQL](https://grafana.com/docs/loki/latest/query/) is a powerful querying language for the Loki logging system. It allows users to filter and aggregate log data to facilitate data analysis and troubleshooting. Here you have the basics to help you get started.
```

**Review by pipo02mix — APPROVED** [review]
some suggestions but LGTM

**Pull request merged**